libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/maneuver/ManeuverViewModel.kt

```diff
@@ -2,76 +2,56 @@ package com.mapbox.navigation.dropin.component.maneuver
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
-import androidx.lifecycle.viewModelScope
 import com.mapbox.navigation.base.formatter.DistanceFormatter
+import com.mapbox.navigation.dropin.component.DropInViewModel
 import com.mapbox.navigation.ui.maneuver.api.MapboxManeuverApi
 import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.onEach
-import kotlinx.coroutines.flow.scan
-import kotlinx.coroutines.flow.stateIn
-import kotlinx.coroutines.launch
 
 @OptIn(ExperimentalCoroutinesApi::class)
-class ManeuverViewModel(
-    private val distanceFormatter: DistanceFormatter
-) : ViewModel() {
+internal class ManeuverViewModel(
+    distanceFormatter: DistanceFormatter
+) : DropInViewModel<ManeuverState, ManeuverAction>(ManeuverState.initial()) {
```

Comment:
```suggestion
) : DropInViewModel<ManeuverState, ManeuverAction>(initialState = ManeuverState.initial()) {
```
NIT: But it helps in reading. Same for all the view models.